### PR TITLE
updated code to match new color scheme naming

### LINKED ISSUE #2738: Colour schemes in UI don't match colour schemes in the code
With the introduction of the new colour schemes we changed the naming convention of schemes in themes.

Previous:
1. Background-1
2. Background-2
3. Accent-1
4. Accent-2
5. Inverse 

New:
1. Scheme 1
2. Scheme 2
3. Scheme 3
4. Scheme 4
5. Scheme 5

------------------------------

## Current behaviour
The theme code still uses the previous naming convention 

Image
Image

## Expected behaviour
The theme code should match the naming convention of the new schemes (ie. scheme 1, 2, 3)

## Possible solutions 

1. Update JSON & CSS variables in Dawn to match Scheme 1-5 nomenclature 
2. Update API to generate readable and consistent naming convention (in place of or in addition to unique ids)
i.e. `scheme-{enum}-{id}` `scheme-6-48uy482638236`

Image

### PR DESCRIPTION
**Pull request by lougoncharenko**
### PR Summary: 
Updated code to match the naming convention of schemes in themes.

### Why are these changes introduced?

Fixes #2738 .

### What approach did you take?
Updated the color schemes names in settings_data.json
Searched through the code to find the words: background-1, background-2, accent-1, accent-2 and inverse and replaced them with their corresponding new scheme names.

### Other considerations
N/A

### Visual impact on existing themes
N/A

### Testing steps/scenarios

- [ ] Create a new color scheme in the themes settings
- [ ] Check settings_data.json file in edit code to ensure the new color scheme follows the same naming convention as the code.

Test all the color schemes in the following sections:
- [ ] announcement bar
- [ ] header
- [ ] image banner
- [ ] featured collection
- [ ] collage
- [ ] video
- [ ] multicolumn
- [ ] blog posts
- [ ] cart 
- [ ] footer
- [ ] ensure the buttons are corresponding with the schemes

Other tests:
- [ ] test in mobile, tablets and desktop
- [ ] test in different browsers

### Demo links

- [Editor](https://admin.shopify.com/store/os2-demo/themes/140247629846/editor)

### Checklist
- [x] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [x] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [x] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [x] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [x] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [x] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [x] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [x] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by kmeleta — COMMENTED** [review]
Looks great! Thanks for picking this one up, it's long over due for us to get around to. If you just rebase and fix the conflicts I'll approve.

File: sections/image-with-text.liquid [hunk 1]
```diff
@@ -242,7 +242,7 @@
       "type": "color_scheme",
       "id": "section_color_scheme",
       "label": "t:sections.all.colors.label",
-      "default": "background-1"
+      "default": "scheme-1"
```
Comment (kmeleta) [inline sections/image-with-text.liquid:245]: Seeing a some stray background-1 default values (in 7 files I believe), presumably from recent changes that were brought in with the rebase. For example, just below here in image-with-text, there's now another color-scheme type setting that needs this same change.

**Review by kmeleta — COMMENTED** [review]
Just need to update a few more instances that probably got brought in with the rebase. Not really noticeable because I believe the color_scheme setting will fallback to the first scheme in the provided default in invalid anyway, but we should clean those up.

File: sections/image-with-text.liquid (hunk 1 above)
Comment (lougoncharenko) [reply sections/image-with-text.liquid:245]: Done! I also double-checked that no additional background-2, accent-1, accent-2 and inverse appeared after the rebase!

**Pull request merged**